src/components/Radio/Radio.stories.tsx

```diff
@@ -0,0 +1,149 @@
+import { Box, HStack, VStack } from "@chakra-ui/react";
+import type { Meta, StoryObj } from "@storybook/react";
+import { withDesign } from "storybook-addon-designs";
+
+import Heading from "../Heading/Heading";
+import Radio from "./Radio";
+
+const meta: Meta<typeof Radio> = {
+  title: "Components/Form Elements/Radio",
+  component: Radio,
+  decorators: [withDesign],
+  argTypes: {
+    className: { control: false },
+    id: { control: false },
+    isChecked: { control: false },
+    isDisabled: {
+      table: { defaultValue: { summary: false } },
+    },
+    isInvalid: {
+      table: { defaultValue: { summary: false } },
+    },
+    isRequired: {
+      table: { defaultValue: { summary: false } },
+    },
+    key: { table: { disable: true } },
+    onChange: { control: false },
+    ref: { table: { disable: true } },
+    showHelperInvalidText: {
+      table: { defaultValue: { summary: true } },
+    },
+    showLabel: {
+      table: { defaultValue: { summary: true } },
+    },
+  },
+};
+
+export default meta;
+type Story = StoryObj<typeof Radio>;
+
+/**
+ * Main Story for the Radio component. This must contains the `args`
+ * and `parameters` properties in this object.
+ */
+export const Controls: Story = {
```

Comment:
To match the other stories, change to `WithControls`.

Reply:
Wait, I thought we said we would remove the word "With" from now on?

Reply:
I guess we shgould talk about this a bit more.  In my head, I imagined we would keep "With" on the first story, but remove it from the others.

Leave it as is for now, but let's talk about this and make a final decision.  Then we can do one final sweep to clean up the story names.